CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Release 2.5.0 19-02-2020
+
+- Auto-generate JWT Audience if no audiences are set in service config
+- Retry after failed token fetch
+- handle disable_auth in oneplatform service config
```

Comment:
don't call oneplatform.  call it endpoint service config

Reply:
Done